test/dice_test.coffee

```diff
@@ -0,0 +1,27 @@
+Helper = require('../node_modules/hubot-test-helper/src/index.coffee')
```

Comment:
You should absolutely not have to do this.

`require('hubot-test-helper'` should be enough, and if it's not something is broken.